src/Gui/DlgGeneral.ui

```diff
@@ -212,6 +213,13 @@ See the FreeCAD Wiki for details about the image.</string>
            </item>
            <item>
             <widget class="QComboBox" name="treeMode"/>
+            <property name="toolTip">
+             <string>Tree View Mode provides the option the option to
+customize the properties panel. The options are:
+1. 'Combiview'
+2. 'TreeView + PropertyView
+3. 'Both'</string>
```

Comment:
@yorikvanhavre These still need explanations. Care to fill them out? BTW, welcome back :)

Reply:
@yorikvanhavre FYI, I wasn't quite done with the PR, hence the `[WIP]` :rofl: